src/coreComponents/physicsSolvers/inducedSeismicity/DieterichSeismicityRate.hpp

```diff
@@ -0,0 +1,85 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+#ifndef GEOS_PHYSICSSOLVERS_INDUCED_SEISMICITY_DIETERICH_SEISMICITY_RATE_HPP_
+#define GEOS_PHYSICSSOLVERS_INDUCED_SEISMICITY_DIETERICH_SEISMICITY_RATE_HPP_
+
+#include "physicsSolvers/inducedSeismicity/SeismicityRateBase.hpp"  // a base class shared by all Laplace solvers
+
+namespace geos
+{
+
+// Like most physics solvers, the Laplace solver derives from a generic SolverBase class.
```

Comment:
I would scrub the file from the Laplace-related comments.

Reply:
Will do.